python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1394,6 +1394,22 @@ class CompareWarpAssembleCoaddConfig(AssembleCoaddConfig):
         dtype=bool,
         default=True,
     )
+    doPrefilterDefects = pexConfig.Field(
+        doc="Ignore artifacts that are mostly covered by the bad pixel mask,"
```

Comment:
What does it mean to "ignore" an artifact? It's not that the pixels' contributions to the coadd are ignored, right? That's covered by other configuration parameters. But it's the pixels' contributions to the count image. Why is that important?